opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/StatsInteractor.java

```diff
@@ -0,0 +1,11 @@
+package org.smartregister.reveal.interactor;
+
+import org.smartregister.reveal.contract.StatsContract;
+
+public class StatsInteractor implements StatsContract.Interactor {
+
+
+    public StatsInteractor() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/app/OpenSRP/opensrp-client-reveal/pullRequest?prid=4587928)